BaseFilter/sections/antiadblock.txt

```diff
@@ -12052,6 +12052,8 @@ doulci.net#@##rightAd
 doulci.net#@##sky_advert
 doulci.net#@##top_ad_game
 doulci.net#@#.rightAd
+! https://github.com/AdguardTeam/AdguardFilters/issues/176764
+stvincenttimes.com#%#//scriptlet('prevent-setTimeout', '#rb-site-access')

```

Comment:
Missed annoyance (subscription)

![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/c0ae51ce-6ead-43ed-a2d7-9b16bcce63d5)


ads

![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/47d7afe5-92b1-4013-bd7a-f2ac672e7135)



Your rule is ok, but I see rules for the same script
https://github.com/AdguardTeam/AdguardFilters/blob/9d3a87917051dce848b501393933248b699b84f7/BaseFilter/sections/antiadblock.txt#L1159-L1167
Please just add a new domain.

Reply:
mmm, yes, i didn't notice that there was similar rule